report/finished/chapters/modeling.tex

```diff
@@ -1,10 +1,60 @@
 \section*{Robotic arm}
 \subsection*{Forward kinematics}
 \subsection*{Inverse kinematics}
+
+
+%%%%%%%%%%%%%%%%%%
+
 \section*{Base}
 
+ The motors will be connected to tracks on the robot and can therefore be modelled as two wheels connected with a rod, as seen in figure \ref{fig:base_math_model}.\\ 
+This means that most of the mathematical modeling is very straight forward given that we can read the encoders from the motors. From the encoders we would be able to get both the length the robot has traveled and more importantly the individual speed each motor rotates with. The speed for the individual motor is of course given by: 
```

Comment:
Deriving the mathematical model is straight forward given.... 

* should thin out words that acts as "fillers"